src/JustEat.StatsD/IStatsDPublisherExtensions.cs

```diff
@@ -134,8 +204,14 @@ public static void Decrement(this IStatsDPublisher publisher, long value, double
         /// <param name="publisher">The <see cref="IStatsDPublisher"/> to publish with.</param>
         /// <param name="value">The value to decrement the counter(s) by.</param>
         /// <param name="sampleRate">The sample rate for the counter(s).</param>
+        /// <param name="tags">The tags to publish with the counter.</param>
         /// <param name="buckets">The bucket(s) to decrement the counter(s) for.</param>
-        public static void Decrement(this IStatsDPublisher publisher, long value, double sampleRate, params string[] buckets)
+        public static void Decrement(
+            this IStatsDPublisher publisher,
+            long value,
+            double sampleRate,
+            IDictionary<string, string?>? tags = null,
```

Comment:
Same here regarding the optional parameter and `params`.

Reply:
Changed with overloads.